.travis.yml

```diff
@@ -34,14 +34,14 @@ python:
 env:
     global:
         - COLUMNS=180
-        - PREVIOUS_NUMPY=1.11.3
+        - PREVIOUS_NUMPY=1.13.3
         - NUMPY_VERSION='stable'
         - ASTROPY_VERSION='stable'
         - MAIN_CMD='python setup.py'
         - SETUP_CMD='test --coverage'
```

Comment:
Do we not want this to just be `test`? We want to only run coverage on one CI test?

Reply:
this is the glory of codecov, we can run it on all of them!

Reply:
Oh it just updates the parts we need to?

Reply:
Yep, it supports uploading multiple reports from different tests.